CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 * [FEATURE] Store-gateway: Allow specific tenants to be enabled or disabled via `-store-gateway.enabled-tenants` or `-store-gateway.disabled-tenants` CLI flags or their corresponding YAML settings. #7653
 * [ENHANCEMENT] Store-gateway: merge series from different blocks concurrently. #7456
 * [ENHANCEMENT] Store-gateway: Add `stage="wait_max_concurrent"` to `cortex_bucket_store_series_request_stage_duration_seconds` which records how long the query had to wait for its turn for `-blocks-storage.bucket-store.max-concurrent`. #7609
+* [ENHANCEMENT] Querier: add histogram to observe wait times when kicking off upstream queries from mergeQuerier. #7209
```

Comment:
Can this be rephrased in terms of what a user thinks about, rather than internal details like `mergeQuerier`? eg. something like "add histogram that measures the wait time before evaluating the query for each tenant in a federated query"

Would also be good to include the name of the metric here.